client/components/userAccount.jsx

```diff
@@ -107,6 +115,63 @@ class UserAccount extends Component {
                   : null}
               </div>
             </div>
+            <div className="column is-half">
+              <p className="title is-4">My Previous Orders</p>
+              <div id="prevUserOrderBox" className="adminBox">
+                {
+                  // console.log(inactiveOrders)
+                  inactiveOrders.length
+                    // eslint-disable-next-line max-len
+                    ? inactiveOrders.filter((order) => order.username === loggedInUser.username).map((order) => (
+                      <div className="box" key={order.inactiveId}>
+                        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+                          <p>
+                            {order.username}
+                            : $
+                            {order.orders.reduce((a, b) => {
+                              // eslint-disable-next-line no-param-reassign
+                              a += (b.quantity * 0.99);
+                              return a;
+                            }, 0)}
```

Comment:
let's do a += ((b.quantity * 99) / 100) like we did with currency elsewhere

Reply:
updated (couldn't remember how we had resolved in the other file, but had made a note to go back)